docs/source/examples.rst

```diff
@@ -6,6 +6,19 @@ Usage Examples
 .. toctree::
    :maxdepth: 1
 
+.. _example_dependencies:
+Example Dependencies
+------------------------
+
+Some examples may use additional dependecies. These dependencies are listed in `examples_requirements.txt <https://github.com/lanl/scico/blob/main/examples/examples_requirements.txt>`_.
+Pip should be used to install these extra requirements except astra which should be installed via conda:
```

Comment:
It looks to me like this PR removes any notion of ASTRA as a dependency for SCICO. I'm uncomfortable with this because (1) ASTRA is needed to run some of the tests and (2) ASTRA-based LinOps are in the API reference without any note saying you must install ASTRA for them to work.

Is there some place where we clarify (even for ourselves) _what_ should work (examples, tests, some other thing) after the user follows _which_ set of install instructions?

Reply:
Agreed, this should be handled better. See also #155.

Reply:
We should have a discussion sometime on issue #155.
In the meantime, should I add a sentence in the contributing section stating that installing the example_requirements is necessary for successfully running the tests?

Reply:
@smajee That, or just conda, given that (I think) it's the only one needed to run the tests. See also my comment about the github action that runs the tests.

Reply:
Added a comment in commit #617e993